agent/engine/common_integ_test.go

```diff
@@ -56,6 +60,43 @@ func createTestTask(arn string) *apitask.Task {
 	}
 }
 
+func setupIntegTestLogs(t *testing.T) string {
+	// Create a directory for storing test logs.
+	testLogDir, err := ioutil.TempDir("", "ecs-integ-test")
+	require.NoError(t, err, "Unable to create directory for storing test logs")
+
+	logger, err := log.LoggerFromConfigAsString(loggerConfigIntegrationTest(testLogDir))
+	if err != nil {
+		t.Log("initialisation failed: %s", err)
+	}
+
+	err = log.ReplaceLogger(logger)
+	if err != nil {
+		t.Log("Unable to replace logger: %v", err)
+	}
+	return testLogDir
+}
+
+func loggerConfigIntegrationTest(logfile string) string {
+	config := `
+	<seelog type="asyncloop" minlevel="debug">
+		<outputs formatid="main">
+			<console />`
+	if logfile != "" {
+		config += `<rollingfile filename="` + logfile + `/ecs-agent-log.log" type="date"
+			 datepattern="2006-01-02-15" archivetype="none" maxrolls="24" />`
+	}
```

Comment:
nit: would using sprintf make this a bit neater?

Reply:
Yes will use that.